dnf-behave-tests/dnf/upgrade.feature

```diff
@@ -174,11 +176,10 @@ Scenario: Upgrade all RPMs from multiple repositories with best=True
         | upgrade       | wget-1:1.19.5-5.fc29.x86_64               |
         | upgrade       | SuperRipper-0:1.2-1.x86_64                |
         | upgrade       | abcde-0:2.9.3-1.fc29.noarch               |
-        | broken        | SuperRipper-0:1.3-1.x86_64                |
```

Comment:
To match the test before the upgrade with best was added can you also add a check for `stderr` at the end?
To make sure dnf5 still reports the problem with best candidate when running with `--no-best`.

Reply:
Done.